src/io/flutter/FlutterBundle.properties

```diff
@@ -123,15 +123,30 @@ flutter.module.create.settings.help.project_type.label=Project type:
 flutter.module.create.settings.help.project_type.description.app=Select an "Application" when building for end users.
 flutter.module.create.settings.help.project_type.description.plugin=Select a "Plugin" when exposing an Android or iOS API for developers.
 flutter.module.create.settings.help.project_type.description.package=Select a "Package" when creating a pure Dart component, like a new Widget.
+flutter.module.create.settings.help.type.application=An "application" is a stand-alone app that can be installed and run on a mobile device.
 flutter.module.create.settings.help.type.package=A "package" is a pure-Dart component, like a new Widget, with no native code.
 flutter.module.create.settings.help.type.plugin=A "plugin" is used to expose an Android or iOS API for developers.
 
+module.wizard.language.name_kotlin=Use Kotlin for Android code (defaults to Java)
+module.wizard.language.name_swift=Use Swift for iOS code (defaults to Objective-C)
```

Comment:
I would suggest, "Use the Swift language for iOS. (defaults to Objective-C)"

Reply:
I'd like a way to indicate that if Kotlin is selected, Java can still be used. That's what the paren phrase is there for but I don't think it's quite right.

Reply:
I don't think we need to explain that; we can assume knowledge of the local platform of someone who is using this capability.